src/sentry/api/serializers/rest_framework/environment.py

```diff
@@ -19,3 +19,7 @@ def to_internal_value(self, data):
         except Environment.DoesNotExist:
             raise ValidationError("Environment is not part of this organization")
         return environment
+
+
+class EnvironmentSerializer(serializers.Serializer):
```

Comment:
can we consolidate `ProjectEnvironmentDetailsPutSerializer` and `EnvironmentSerializer`? we might be able to pass this for the request schema for `ProjectEnvironmentDetailsEndpoint::Put` above